include/library/spdm_crypt_lib.h

```diff
@@ -875,6 +875,9 @@ bool libspdm_get_random_number(size_t size, uint8_t *rand);
 
 #if LIBSPDM_CERT_PARSE_SUPPORT
 
+#define SPDM_ALIAS_CERT_MODEL   0
+#define SPDM_DEVICE_CERT_MODEL  1
```

Comment:
why put here?

Reply:
Now remove it, use the definitions(SPDM_CERTIFICATE_INFO_CERT_MODEL_DEVICE_CERT ..  ) that is already defined in the [spdm.h](https://github.com/DMTF/libspdm/blob/22a9055bf545ed225ad7648ded1decddc47b29a9/include/industry_standard/spdm.h#L493)